bundles/rhsso-operator/bundles.yaml

```diff
@@ -0,0 +1,7 @@
+bundles:
+    - name: keycloak-operator.v18.0.0
+      image: quay.io/integreatly/rhsso-bundle:18.0.0
+    - name: rhsso-operator.7.5.2-9
+      image: quay.io/integreatly/rhsso-bundle:7.5.2-9
+    - name: rhsso-operator.7.6.0-4
+      image: quay.io/integreatly/rhsso-bundle:7.6.0-4
```

Comment:
@valerymo should this not be the official registry bundle instead?

Reply:
@MStokluska Thanks very much for comment. I will try use official bundle for 7.6.0-4, and will provide update.  Thank you!

Reply:
@MStokluska Fix done. Official registry bundle is used for 7.6.0-4. Attaching generated Index content
[index_rhsso7.6.0-4.txt](https://github.com/integr8ly/integreatly-operator/files/9399679/index_rhsso7.6.0-4.txt)
.
Thank you for review and comment